Cache/BlockCacheManager.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the Sonata package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Cache;
+
+use Doctrine\Common\Util\ClassUtils;
+use Sonata\BlockBundle\Block\BlockContextManagerInterface;
+use Sonata\BlockBundle\Model\BlockInterface;
+use Sonata\Cache\CacheManagerInterface;
+
+/**
+ * Class BlockCacheManager.
```

Comment:
This comment is useless. We already now this is the BlockCacheManager class. :-)

Just set your `@author` tag if you have nothing to tell more. :+1: